src/app/shared/ubs-footer/ubs-footer.component.scss

```diff
@@ -79,7 +79,7 @@
         }
 
         button:hover {
-          color: beige;
+          color: var(--ubs-button-beige);
```

Comment:
![Screenshot 1](https://user-images.githubusercontent.com/51752239/167104089-bd8b76b9-5ee6-4b6a-be31-dd3fd6d351c7.png)
Why button in hover has beige color, in the mockups the color of text doesn't change when you hover the button?

Reply:
the button which has been given as example by tester in my task had beige on hover in styles, and I haven't checked our mockups. thank you for noticing, I'll correct it for both buttons